dynatrace/api/builtin/hyperscalerauthentication/connections/azure/authentication/service.go

```diff
@@ -41,14 +41,35 @@ type service struct {
 }
 
 func (me *service) List(ctx context.Context) (api.Stubs, error) {
-	return me.connService.List(ctx)
+	stubs, err := me.connService.List(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	var filteredStubs api.Stubs
+	for _, stub := range stubs {
+		connValue := azureconnection_settings.Settings{}
+		if err := me.connService.Get(ctx, stub.ID, &connValue); err != nil {
+			return nil, err
+		}
```

Comment:
There isn't any easy way for fetching all in one call without touching a lot of code I guess?